acquisitions/src/main/resources/thunderjet/mod-finance/features/create-inactive-budget.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Create inactive budget
```

Comment:
user story number can be added in comment

Reply:
I meant to add that and forgot !